composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "jimdo/prometheus_client_php",
+    "name": "imj/prometheus_client_php",
```

Comment:
Follow the way you mentioned, I encountered a similar problem like this:  https://www.reddit.com/r/PHP/comments/2ctjb8/is_it_possible_to_use_composerpackagist_with/cjivn7e/ , so I will temporarily maintain tag on other branches.